prime-router/src/testIntegration/kotlin/datatests/TranslationTests.kt

```diff
@@ -375,6 +387,23 @@ class TranslationTests {
                 } else null
             }
 
+            if (!config.conditionFiler.isNullOrBlank()) {
+                if (config.conditionFiler.equals("pruneUnwanted")) {
```

Comment:
Oh just saw that SonarCloud prefers usage of `==` instead of `.equals`. Hard habit to break coming from java, but if it helps keep SonarCloud quiet, it's probably worth making the change.